### DISCUSSION TITLE
Why every time I do a fetch, it gets the old articles again!?

### DISCUSSION THREAD

**Discussion by Tony20221** [martinrotter/rssguard#1437] [Q&A]
Why every time I do a fetch, it gets the old articles again!? It's supposed to get new ones.

Is that because I cleaned the database? How else does one clear the deleted articles from the screen?

**Comment by martinrotter** [comment]
Hi Tony.

This is essentially bug report, so report is as as a bug and in "Issues" section and provide application debug log, so that I can actually see, where the problem is.

Also, when creating the bug report, provide full URL of the feed which is having the problem.

When you "clear database" you essential prune (completely physically remove) the articles from the DB, so naturally, when you fetch the feed, they are fetched again. To remove some articles, simply select them and then hit DELETE. They will be moved to recycle bin.

You can then empty your recycle bin in which case articles will be permanently "hidden" and will not be refetched again.

https://rssguard.readthedocs.io/en/stable/contrib/bugs.html
